lintro/ai/review/chunker/workflow_scripts.py

```
@@ -0,0 +1,923 @@
+                return True
+        shell_cwd = _shell_cwd_after_line(line=line, cwd=shell_cwd)
+    return block_style == "folded" and _folded_block_references_script()
+
+
+def _workflow_post_image_text(*, workflow_diff: str) -> str:
+    """Return the post-change workflow text represented by diff hunks."""
+    lines: list[str] = []
+    for line in workflow_diff.splitlines():
+        if line.startswith(("+++", "---", "diff --git", "index ", "@@")):
+            continue
+        if line.startswith("+") or line.startswith(" "):
```

Ruff (0.15.20)

[warning] Call `startswith` once with a `tuple`

Merge into a single `startswith` call

(PIE810)